bft/testers/sqlite/runner.py

```diff
@@ -54,17 +52,20 @@ def run_sql_case(self, case: Case, mapping: SqlMapping) -> SqlCaseResult:
             if mapping.aggregate:
                 arg_names = [arg_names[0]]
             joined_arg_names = ",".join(arg_names)
-            arg_vals = ",".join(
-                [
-                    "'" + literal_to_str(arg) + "'"
-                    if arg.type == "string" and arg.value is not None
-                    else literal_to_str(arg)
-                    for arg in case.args
-                ]
-            )
+            arg_vals = []
+            for arg in case.args:
+                if arg.type == "string" and arg.value is not None:
+                    arg_vals.append("'" + literal_to_str(arg.value) + "'")
+                elif mapping.aggregate:
+                    for value in arg.value:
+                        arg_vals.append(literal_to_str(value))
+                else:
+                    arg_vals.append(literal_to_str(arg.value))
+
+            arg_vals = ",".join(arg_vals)
             if mapping.aggregate:
                 arg_vals_list = ", ".join(f"({val})" for val in arg_vals.split(","))
-                if arg_vals != "[]":
+                if len(arg_vals):
```

Comment:
Python typically just a boolean check here without the length (i.e.  `if arg_vals:`).  It would also exclude None this way which is extra but it shouldn't happen.

Reply:
made the change, thanks.